sg_otio/sg_cut_track_writer.py

```diff
@@ -1013,7 +1018,9 @@ def _get_cut_entities(self, entity_type, entity_id):
             sg_linked_entity = self._sg.find_one(
                 entity_type,
                 [["id", "is", entity_id]],
-                ["project", "code"]
+                # We might need the absolute cut order of the linked entity later, it's better
+                # to retrieve it now.
+                ["project", "code", _ABSOLUTE_CUT_ORDER_FIELD]
```

Comment:
Any worry about using a constant here and `sfg.absolute_cut_order` above? May be a comment would be useful?

Reply:
it's kind of a dumb chicken and egg situation: you need the linked entity to get the shot fields config to get the absolute cut order field... So if I did it "properly", it would require two SG queries instead of one.

Reply:
I added a comment.